.github/workflows/build_linux.yml

```diff
@@ -38,7 +38,7 @@ on:
 jobs:
   build_desktop:
     name: build-linux-unity${{inputs.unity_version}}-CPP${{ inputs.firebase_cpp_sdk_version }}
-    runs-on: ubuntu-20.04
+    runs-on: ubuntu-latest
```

Comment:
Is this change intentional? Remember we need to ping to ubuntu 20

Reply:
Yes. 
I believe github will remove ubuntu 20 runner in the future. Then, we still need to solve the issue.
This PR is to fix the installation issue on newer ubuntu runners, so that we don't have to ping to ubuntu 20.

Reply:
Unity does not support Ubuntu 22.04

Reply:
See forum post for details https://forum.unity.com/threads/running-unity-on-ubuntu-22-04.1284083/

I've already tried the workaround in my installation scripts but cannot get around the fuse error

Reply:
Thanks for the info. 
Yeah, I also tried the workaround and it didn't work.
But, we are building SDK here and not launching Editor.
So, it seems working in my CI:
```
I1129 21:42:55.199143 140596922395520 unity_installer.py:343] run_with_retry: xvfb-run unityhub --headless install --version 2020.3.40f1 --changeset ba48d4efcef1 (attempt 1 of 3)
I1129 21:46:01.864848 140596922395520 unity_installer.py:346] cmd stdout: This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). The promise rejected with the reason:
Error: Unknown or unsupported transport “disabled” for address “disabled:”
This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). The promise rejected with the reason:
...
Progress:
[Unity 2020.3.40f1] downloading 98.77%
Progress:
[Unity 2020.3.40f1] downloading 100.00%
Progress:
[Unity 2020.3.40f1] finished downloading.
Progress:
[Unity 2020.3.40f1] finished downloading.
Progress:
[Unity 2020.3.40f1] queued for install.
Progress:
[Unity 2020.3.40f1] queued for install.
Progress:
[Unity 2020.3.40f1] validating installation...
Progress:
[Unity 2020.3.40f1] validating installation...
Progress:
[Unity 2020.3.40f1] installing...
Progress:
[Unity 2020.3.40f1] installed successfully.
All Tasks Completed Successfully.
Progress:
[Unity 2020.3.40f1] installed successfully.
All Tasks Completed Successfully.
```

Reply:
You guys don't do and validation and testing in editor?

Reply:
We do. But not on Ubuntu runners.